### PR TITLE
increased tests and coverages and some re-organizations

### PR DESCRIPTION
**Pull request by meltedspork**
Added lots more of tests and coverages 👍 

also re-organizated some stuff around where made most sense.

### CODE REVIEW

File: mocks/jwt-decode.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default (tokenAsDate) => {
```
Comment (kratsg) [inline mocks/jwt-decode.js:7]: what is this mocking doing? A bit confused here. More comments...

File: src/utils/auth-constant.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (kratsg) [inline src/utils/auth-constant.js]: this should go in directory w/ all other constants, no?

File: tests/utils/AuthService.test.js [hunk 3]
```diff
@@ -0,0 +1,387 @@
+import AuthService from '../../src/utils/AuthService';
+import AUTH0 from '../../src/utils/auth-web';
+
+import {
+  ID_TOKEN,
+  ACCESS_TOKEN,
+  PROFILE,
+} from '../../src/utils//auth-constant';
```
Comment (kratsg) [inline tests/utils/AuthService.test.js]: fix path

**Review by kratsg — CHANGES_REQUESTED** [review]

File: mocks/jwt-decode.js (hunk 1 above)
Comment (meltedspork) [reply mocks/jwt-decode.js:7]: Ah, my bad. I should have done that already. Added more comments just now.

**Review by kratsg — APPROVED** [review]

**Pull request merged**